core/src/main/java/hudson/cli/handlers/ViewOptionHandler.java

```diff
@@ -81,22 +82,18 @@ public int parseArguments(Parameters params) throws CmdLineException {
      * Note: Personal user's views aren't supported now.
      *
      * @param name A view name
-     * @return The {@link View} instance. Null if {@link Jenkins#getInstance()} returns null
-     *         sor user doesn't have a READ permission.
-     * @throws CmdLineException
-     *      If view isn't found or an un-expected error occurred
+     * @return The {@link View} instance. Null if name is empty string
+     * @throws IllegalArgumentException - If view isn't found
+     * @throws IllegalStateException - If cannot get active Jenkins instance or view can't conain views
+     * @throws AccessDeniedException - If user doens't have a READ permission for the view
      * @since TODO
      */
     @CheckForNull
     public View getView(final String name) throws CmdLineException {
```

Comment:
`CmdLineException` is now unused in the method. According to the [1] it only break source compatibility but not binary one. Can we remove it?

[1] https://wiki.eclipse.org/Evolving_Java-based_APIs_2